xen-bugtool

```diff
@@ -268,7 +266,6 @@ BIOSDEVNAME = 'biosdevname'
 BRCTL = 'brctl'
 CHKCONFIG = 'chkconfig'
 CHRONYC = 'chronyc'
-DCBTOOL = 'dcbtool'
```

Comment:
Only as a side note, for reference (obsolete) as this PR is not needed:

This package is not a file from `fcoe-utils`:
```py
rpm -qf /usr/sbin/dcbtool
lldpad-1.0.1-10.xs8.x86_64
```